aim/web/ui/src/components/ExportPreview/config.ts

```diff
@@ -0,0 +1,25 @@
+import { IPreviewBounds } from './ExportPreview.d';
+
+const previewBounds: IPreviewBounds = {
```

Comment:
Maybe we can keep constants names with SCREAMING_SNAKE_CASE
https://en.wikipedia.org/wiki/Snake_case

Reply:
thanks,  done